api/app/signals/apps/users/v1/serializers/roles.py

```diff
@@ -0,0 +1,30 @@
+from datapunt_api.rest import HALSerializer
+from datapunt_api.serializers import DisplayField
+from django.contrib.auth.models import Group, Permission
+from rest_framework import serializers
+
+from signals.apps.users.v1.serializers.permissions import PermissionSerializer
+
+
+def _get_permissions_queryset():
+    return Permission.objects.all()
+
+
+class RoleSerializer(HALSerializer):
+    _display = DisplayField()
+    permissions = PermissionSerializer(many=True, required=False, read_only=True)
+    permission_ids = serializers.PrimaryKeyRelatedField(
+        many=True, required=False, read_only=False, write_only=True,
+        queryset=_get_permissions_queryset(), source='permissions'
+    )
```

Comment:
please explain

Reply:
This is done so that you can use a list of id's to add to the role (or other objects I use this). It is now possible when creating to just say `permission_ids: [1,2,3]` and this will result in adding the permissions with the provided id's to the role.

I will explain this to the frontend